src/UsersView/Users.style.ts

```diff
@@ -0,0 +1,41 @@
+import { StyleSheet } from "react-native"
+
+export const styles = StyleSheet.create({
```

Comment:
Can add a model here too like so:

export const styles = StyleSheet.create<Style>

and specify the model like so:

interface Style {
    backgroundMidle: ViewStyle
    textAlightCenter: TextStyle
}